### PR TITLE
[Tracing][PHP][.NET] Review W3C trace context documentation for PHP and .NET

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do?
Change documentation around W3C trace context propagation for PHP and .NET now that the changes have been released.

### Motivation
Changes have been released

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/library_config/php.md [hunk 1]
```diff
@@ -305,27 +305,30 @@ The IP header to be used for client IP collection, for example: `x-forwarded-for
...
+
+  - [tracecontext][10]
+  - [b3multi][7]
   - [B3 single header][8]
   - Datadog
 
 `DD_TRACE_PROPAGATION_STYLE_EXTRACT`
-: **INI**: `datadog.trace.propagation_style_extract`<br>
+: **INI**: `tracecontext,Datadog,b3multi,B3 single header`<br>
 **Default**: `Datadog,B3,B3 single header`<br>
```
Comment (bwoebi) [inline content/en/tracing/trace_collection/library_config/php.md:327]: Wrong line replaced.

File: content/en/tracing/trace_collection/library_config/php.md [hunk 2]
```diff
@@ -422,6 +425,28 @@ Note that `DD_TRACE_RESOURCE_URI_MAPPING_INCOMING` applies to only incoming requ
...
+- B3 Single Header: `B3 single header`
+
+You can use the following environment variables to configure injection and extraction styles. For instance:
+
+- `DD_TRACE_PROPAGATION_STYLE_INJECT=Datadog,tracecontext`
+- `DD_TRACE_PROPAGATION_STYLE_EXTRACT=Datadog,tracecontext`
+
+The environment variable values are comma-separated lists of header styles enabled for injection or extraction. By default, only the `Datadog` injection style is enabled.
+
+If multiple extraction styles are enabled, the extraction attempt is completed in order of configured styles, and uses the first successful extracted value.
```
Comment (bwoebi) [inline content/en/tracing/trace_collection/library_config/php.md:448]: That's wrong, it's hardcoded in code. (tracecontext has prio, then datadog, then b3).

File: content/en/tracing/trace_collection/library_config/php.md [hunk 3]
```diff
@@ -422,6 +425,28 @@ Note that `DD_TRACE_RESOURCE_URI_MAPPING_INCOMING` applies to only incoming requ
 When [`open_basedir`][9] setting is used, then `/opt/datadog-php` should be added to the list of allowed directories.
 When the application runs in a docker container, the path `/proc/self` should also be added to the list of allowed directories.
 
+### Headers extraction and injection
+
+The Datadog APM Tracer supports [B3][7] and [W3C][10] headers extraction and injection for distributed tracing.
+
+You can configure injection and extraction styles for distributed headers.
+
+The .NET Tracer supports the following styles:
```
Comment (bwoebi) [inline content/en/tracing/trace_collection/library_config/php.md:434]: Copypasta (.NET)

**Comment by bwoebi** [comment]
Is the PHP doc intentionally missing `DD_TRACE_PROPAGATION_STYLE` (defaults for inject and extract if they aren't defined)?

File: content/en/tracing/trace_collection/library_config/php.md (hunk 2 above)
Comment (pierotibou) [reply content/en/tracing/trace_collection/library_config/php.md:448]: Hmm, wait. The RFC says:

`When a service-entry span is created, headers from the incoming request are inspected for the specified distributed tracing styles in the order that they were provided. This applies to HTTP requests and, if applicable instrumentation exists for a tracing library, gRPC requests.`

Am I missing something?

File: content/en/tracing/trace_collection/library_config/php.md [hunk 4]
```diff
@@ -422,6 +425,28 @@ Note that `DD_TRACE_RESOURCE_URI_MAPPING_INCOMING` applies to only incoming requ
...
+- W3C: `tracecontext`
+- B3 Multi Header: `b3multi` (`B3` is deprecated)
+- B3 Single Header: `B3 single header`
+
+You can use the following environment variables to configure injection and extraction styles. For instance:
+
+- `DD_TRACE_PROPAGATION_STYLE_INJECT=Datadog,tracecontext`
+- `DD_TRACE_PROPAGATION_STYLE_EXTRACT=Datadog,tracecontext`
+
+The environment variable values are comma-separated lists of header styles enabled for injection or extraction. By default, only the `Datadog` injection style is enabled.
```
[the remark this answers is not in the document]
Comment (pierotibou) [reply content/en/tracing/trace_collection/library_config/php.md:446]: Thanks

File: content/en/tracing/trace_collection/library_config/php.md (hunk 3 above)
Comment (pierotibou) [reply content/en/tracing/trace_collection/library_config/php.md:434]: :oops:

File: content/en/tracing/trace_collection/library_config/php.md (hunk 1 above)
Comment (pierotibou) [reply content/en/tracing/trace_collection/library_config/php.md:327]: Thanks

File: content/en/tracing/trace_collection/library_config/php.md [hunk 5]
```diff
@@ -432,6 +435,28 @@ Note that `DD_TRACE_RESOURCE_URI_MAPPING_INCOMING` applies to only incoming requ
...
+- B3 Single Header: `B3 single header`
+
+You can use the following environment variables to configure injection and extraction styles. For instance:
+
+- `DD_TRACE_PROPAGATION_STYLE_INJECT=Datadog,tracecontext`
+- `DD_TRACE_PROPAGATION_STYLE_EXTRACT=Datadog,tracecontext`
+
+The environment variable values are comma-separated lists of header styles enabled for injection or extraction. By default, only the `tracecontext` and `Datadog` injection styles are enabled.
+
+If multiple extraction styles are enabled, the extraction attempt is completed with the following priorities: tracecontext has priority, then datadog, then b3.
```
Comment (cswatt) [inline content/en/tracing/trace_collection/library_config/php.md:458]:
```suggestion
If multiple extraction styles are enabled, the extraction attempt is completed with the following priorities: `tracecontext` has priority, then `Datadog`, then B3.
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**